index.html

```diff
@@ -431,32 +431,23 @@
 
                 <p>The target object at which the event is fired is determined as follows:
                 <ul>
-                    <li>If the <a>pointer capture target override</a> has been set for the pointer, 
-                        <ul>
-                            <li>Set the <code>relatedTarget</code> attribute of the event to <code>null</code>.</li>
-                            <li>Set the target to the <a>pointer capture target override</a> object.</li>
-                        </ul>
-                    </li>
+                    <li>If the <a>pointer capture target override</a> has been set for the pointer, set the target to <a>pointer capture target override</a> object.</li>
                     <li>Otherwise, set the target to the object returned by normal hit test mechanisms (out of scope for this specification).</li>
                 </ul>
 
                 <p>If this is a <code>pointerdown</code> event, the associated device is a direct manipulation device and the target is an <code>Element</code>, then <a href="setting-pointer-capture">set pointer capture</a> for this <code>pointerId</code> to the target element as described in <a>implicit pointer capture</a>.
                 
                 <p>Fire the event to the determined target.
 
+                <div class="note">Using the <a>pointer capture target override</a> as the target instead of the normal hit-test result may fire some boundary events. This is the same as the pointer leaving its previous target and entering this new capturing target and if they are different targets boundary events should be dispatched first. When the capture is released the same scenario may happen as the pointer is leaving the capturing node and entering the hit-test target.</div>
```

Comment:
This is a little hand-wavy (it's non-normative text but it describes behavior that isn't captured by the normative spec text outside of notes).  As you know I'm OK with being a little imprecise (as least for now) as long as we have tests that validate the intended precise behavior.

One simple way to improve this may be to change the sections that describe the boundary events.  They currently say "when a pointing device is moved into the hit test boundaries of ..." etc.  If instead they said something like "when the target for the events for a given pointerId changes to ...".  I.e. boundary events are generated as a result of changes in event targets within a per-pointer event stream, not as a result of movement around "hit-test boundaries".

Even better, this could be encoded precisely in the firing algorithm (eg. by defining a "last target per pointer" map).  But that's probably a fair bit more work.  Maybe file a spec bug for that to explore in detail later?